apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/CopyObjectOptions.cs

```diff
@@ -110,6 +110,11 @@ public sealed class CopyObjectOptions
         /// </summary>
         public EncryptionKey SourceEncryptionKey { get; set; }
 
+        /// <summary>
+        /// The name of the Cloud KMS key to use to encrypt the new object. If this is null and customer-supplied encryption is not being used,
+        /// the bucket encryption defaults will be used to determine the encryption for the object.
+        /// </summary>
```

Comment:
Should there be remarks here and on `EncryptionKey` to say how they shouldn't both be specified? (same question for UploadObjectOptions.cs)

Reply:
Quite possibly. Although it's fine to specify that the EncryptionKey is None. I'll wait until the semantics are settled, then add another commit for that (or another PR if it's post-merge).